packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -229,21 +460,34 @@ class CupertinoSliverNavigationBar extends StatelessWidget {
   /// any [GlobalKey]s, and that it not rely on maintaining state (for example,
   /// animations will not survive the transition from one location to the other,
   /// and may in fact be visible in two places at once during the transition).
+  ///
+  /// If null and [automaticallyImplyTitle] is true, an appropriate [Text]
+  /// title will be created if the current route is a [CupertinoPageRoute] and
+  /// has a `title`.
+  ///
+  /// This parameter must either be non-null or [automaticallyImplyTitle] must
+  /// be true and the route has a title.
```

Comment:
"This parameter must either be non-null or the route must have a title ([...]) and [automaticallyImplyTitle] must be true."
where "[...]" is a cross-reference to the relevant thing in CupertinoPageRoute or wherever.